include/s3select_json_parser.h

```diff
@@ -369,13 +373,14 @@ void dec_key()
   }
 }
 
-void new_value(s3selectEngine::value& v,size_t json_index)
+void new_value(s3selectEngine::value& v,size_t json_index, json_key_value_t key_path_value)
 {
   DBG
 
   if(is_on_final_state())
   {
     (*m_exact_match_cb)(v, json_index);
+    (*m_key_path_operation_cb)(key_path_value);
```

Comment:
upon s3select has an indication for JSON output format, it enables to skip this operation ` (*m_key_path_operation_cb)(key_path_value);`